tests/test_crd.py

```diff
@@ -0,0 +1,81 @@
+"""Unit tests for crd.py."""
+
+import os
+import tempfile
+
+import numpy as np
+
+from simSPI import crd
+
+
+def test_write_crd_file():
+    """Test whether function creates and writes to file on given path."""
+    temp_dir = tempfile.TemporaryDirectory()
+    temp_file = temp_dir.name + "/crd.txt"
+
+    assert not os.path.isfile(temp_file)
+    crd.write_crd_file(numpart=20, crd_file=temp_file)
+    assert os.path.isfile(temp_file)
+    crd.write_crd_file(numpart=20, crd_file=temp_file)
+    assert os.path.isfile(temp_file)
+
+
+def test_get_rotlist():
+    """Test whether randomly generated rotlist is of correct length."""
+    numpart = 10
+    assert len(crd.get_rotlist(numpart)) == len(range(0, numpart + 1))
```

Comment:
Putting `== numpart` directly would be more readable?

also: numpart -> n_particles